### ISSUE TITLE
Select: <svg> elements with an img role must have an alternative text -[a11y issue]

### ISSUE THREAD

**Issue by techie-khushdeep** [SAP/ui5-webcomponents#10986]
### Describe the bug

Hi,
We are using select component in our application. We are getting a11y issue in axe dev tools as well as jest test.
**A11y issue**: Ensure <svg> elements with an img, graphics-document or graphics-symbol role have an accessible text.

Issue screenshot:

![Image](https://github.com/user-attachments/assets/e3609a56-39d4-4d9d-91ff-4368d9e1e2b0)

**official component url**: https://sap.github.io/ui5-webcomponents-react/v2/?path=/docs/inputs-select--docs

### Isolated Example

https://stackblitz.com/edit/github-h2thrpbd-dhjhe67o?file=src%2FApp.tsx

### Reproduction steps

1. Run the application using https://stackblitz.com/edit/github-h2thrpbd-dhjhe67o?file=src%2FApp.tsx
2. Do the scan using axe browser extension and check the a11y issue.

### Expected Behaviour

There should be no a11y issue for component.

### UI5 Web Components for React Version

2.5.0

### UI5 Web Components Version

2.5.0

### Browser

Chrome

### Relevant log output

```shell

```

### Organization

SAP

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by Todor-ads** [comment]
Hello @SAP/ui5-webcomponents-topic-p  ,
could you please check this issue?
Thanks,
Todor

**Comment by yanaminkova** [comment]
Hello @techie-khushdeep,


The issue was fixed with the following change: https://github.com/SAP/ui5-webcomponents/pull/10969
and the PR is included in version v2.9.0-rc.0

You could test it on the following[ link.](https://sap.github.io/ui5-webcomponents/nightly/play/#eyJpbmRleC5odG1sIjp7Im5hbWUiOiJpbmRleC5odG1sIiwiY29udGVudCI6IjwhLS0gcGxheWdyb3VuZC1mb2xkIC0tPlxuPCFET0NUWVBFIGh0bWw-XG48aHRtbCBsYW5nPVwiZW5cIj5cblxuPGhlYWQ-XG4gICAgXG4gICAgPHN0eWxlPlxuICAgICAgKjpub3QoOmRlZmluZWQpIHtcbiAgICAgICAgZGlzcGxheTogbm9uZTtcbiAgICAgIH1cblxuICAgIGh0bWwge1xuICAgICAgZm9yY2VkLWNvbG9yLWFkanVzdDogbm9uZTtcbiAgICB9XG4gICAgPC9zdHlsZT5cblxuICAgIDxtZXRhIGNoYXJzZXQ9XCJVVEYtOFwiPlxuICAgIDxtZXRhIG5hbWU9XCJ2aWV3cG9ydFwiIGNvbnRlbnQ9XCJ3aWR0aD1kZXZpY2Utd2lkdGgsIGluaXRpYWwtc2NhbGU9MS4wXCI-XG4gICAgPHRpdGxlPlNhbXBsZTwvdGl0bGU-XG48L2hlYWQ-XG48Ym9keSBzdHlsZT1cImJhY2tncm91bmQtY29sb3I6IHZhcigtLXNhcEJhY2tncm91bmRDb2xvcik7IGhlaWdodDogMjUwcHhcIj5cbjwhLS0gcGxheWdyb3VuZC1mb2xkLWVuZCAtLT5cblxuXG4gICAgPHVpNS1zZWxlY3Q-XG4gICAgICAgIDx1aTUtb3B0aW9uIGljb249XCJsYXB0b3BcIj5EZXNrdG9wPC91aTUtb3B0aW9uPlxuICAgICAgICA8dWk1LW9wdGlvbiBpY29uPVwiaXBhZFwiIHNlbGVjdGVkPlRhYmxldDwvdWk1LW9wdGlvbj5cbiAgICAgICAgPHVpNS1vcHRpb24gaWNvbj1cImlwaG9uZVwiPlBob25lPC91aTUtb3B0aW9uPlxuICAgIDwvdWk1LXNlbGVjdD5cblxuPCEtLSBwbGF5Z3JvdW5kLWZvbGQgLS0-XG4gICAgPHNjcmlwdCB0eXBlPVwibW9kdWxlXCIgc3JjPVwibWFpbi5qc1wiPjwvc2NyaXB0PlxuPC9ib2R5PlxuPC9odG1sPlxuPCEtLSBwbGF5Z3JvdW5kLWZvbGQtZW5kIC0tPlxuXG4ifSwibWFpbi5qcyI6eyJuYW1lIjoibWFpbi5qcyIsImNvbnRlbnQiOiIvKiBwbGF5Z3JvdW5kLWhpZGUgKi9cbmltcG9ydCBcIi4vcGxheWdyb3VuZC1zdXBwb3J0LmpzXCI7XG4vKiBwbGF5Z3JvdW5kLWhpZGUtZW5kICovXG5pbXBvcnQgXCJAdWk1L3dlYmNvbXBvbmVudHMvZGlzdC9TZWxlY3QuanNcIjtcbmltcG9ydCBcIkB1aTUvd2ViY29tcG9uZW50cy9kaXN0L09wdGlvbi5qc1wiO1xuaW1wb3J0IFwiQHVpNS93ZWJjb21wb25lbnRzLWljb25zL2Rpc3QvaXBhZC5qc1wiO1xuaW1wb3J0IFwiQHVpNS93ZWJjb21wb25lbnRzLWljb25zL2Rpc3QvaXBob25lLmpzXCI7XG5pbXBvcnQgXCJAdWk1L3dlYmNvbXBvbmVudHMtaWNvbnMvZGlzdC9sYXB0b3AuanNcIjsifX0)

Best regards,
Yana
